apps/admin-x-activitypub/package.json

```diff
@@ -70,5 +70,8 @@
         ]
       }
     }
+  },
+  "dependencies": {
+    "use-debounce": "10.0.3"
   }
```

Comment:
@allouis @djordjevlais Sus that this is the only non dev dependency 🤔 Have I misunderstood how we are using this here (i.e we just stick everything in `devDependencies`)?

Reply:
Yeah kinda weird, but I guess because we build a bundle with all the deps, we technically only needed them during build time which will have devDeps. Happy for them to be moved, I think it's weird.

Reply:
👍 I've moved anything thats strictly not development related (i.e testing shtuff) to `dependencies`